sale_order_type/models/sale_order_type.py

```diff
@@ -65,3 +65,6 @@ def default_invoice_state(self):
     invoice_state = fields.Selection(
         selection='_get_selection_invoice_state', string='Invoice Control',
         required=True, default=default_invoice_state)
+    company_id = fields.Many2one(
```

Comment:
I'm not sure if this will work due to a bug/design decision in the ORM. If you change the warehouse_id and save, is this field correctly updated?